src/nvim/msgpack_rpc/channel.c

```diff
@@ -251,13 +259,10 @@ Object channel_send_call(uint64_t id,
     return NIL;
```

Comment:
Oh, sorry. I annotated the wrong return. I meant the one withing `if (frame.errored)`, as I asked at the beginning in the main conversation thread.